src/send/hooks.ts

```diff
@@ -49,6 +50,7 @@ export function useMergedSearchRecipients(onSearch: (searchQuery: string) => voi
 
   const throttledSearch = throttle((searchInput: string) => {
     onSearch(searchInput)
+    searchQueryRef.current = searchInput
```

Comment:
seems like searchQueryRef and searchQuery state are very similar. did you try combining them?

Reply:
We can drop the ref and do something like `if (searchQuery === searchInput) return` at the start of this function right?

Reply:
Fixed in 091f1fae818fdf92b744fbd0e16aa4effba424ca using @satish-ravi suggestion; seems a lot cleaner and more contained.